### PR TITLE
Support for nested imports

### PR DESCRIPTION
**Pull request by abhishekrb19**
Addresses: https://github.com/dropbox/mypy-protobuf/issues/38

- Adds in nested imports support. This is really a one liner change since proto nested imports would translate to importing the corresponding package and referencing the inner classes in that package. 
- Adds test for protos and expected pyi for nested imports (enums and messages).

@nipunn1313 , please let me know if there is something else that needs to be added.

### CODE REVIEW

**Comment by nipunn1313** [comment]
Hi! Thanks for the PR @abhishekrb19  - I will take a look

File: python/protoc-gen-mypy [hunk 1]
```diff
@@ -91,9 +91,7 @@ class PkgWriter(object):
                 remains = ".".join(split[i + 1:])
                 if not remains:
                     return import_name
-                raise AssertionError("Don't support nested imports yet")
-                # return new_nested_import(import_name, remains)
-
+                return import_name + "." + remains
```
Comment (nipunn1313) [inline python/protoc-gen-mypy]: Could you please leave a comment somewhere around here explaining what "remains" refers to - to make it clear to a peruser that it refers to a direct import of a nested enum or message from another package.

**Comment by nipunn1313** [comment]
Hi @abhishekrb19 - this diff looks great! I added one small request around adding a comment for maintainability, but otherwise, I'm happy to approve!

Thanks for doing the nice test case - that really helped me review more easily.

Could you additionally take a look at Dropbox's CLA as well? Thanks.
https://opensource.dropbox.com/cla/

File: python/protoc-gen-mypy (hunk 1 above)
Comment (abhishekrb19) [reply python/protoc-gen-mypy]: @nipunn1313 , done.

**Comment by abhishekrb19** [comment]
@nipunn1313 , thanks for the review. I filled out the CLA and added a comment. 
Apologies for the delay.

**Comment by nipunn1313** [comment]
Thanks!

**Pull request merged**